stream/src/main/scala/org/apache/pekko/stream/impl/fusing/Switch.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
brand new source files with no copying from existing source files should have the standard Apache source header not this one.

https://github.com/apache/pekko/blob/main/project/AddMetaInfLicenseFiles.scala#L1-L16